query/benchmarker.go

```diff
@@ -22,45 +22,46 @@ const (
 // BenchmarkRunner contains the common components for running a query benchmarking
 // program against a database.
 type BenchmarkRunner struct {
-	sp      *statProcessor
-	scanner *scanner
-	c       chan Query
-
+	// flag fields
 	dbName         string
-	workers        uint
 	limit          uint64
 	memProfile     string
+	workers        uint
 	printResponses bool
 	debug          int
 	fileName       string
 
+	// non-flag fields
 	br            *bufio.Reader
+	statProcessor *statProcessor
```

Comment:
I prefer the shorter names of `sp` and `c`, pretty common golang-ism to keep them short.

Reply:
Ok, agreed
Summary: updated